csrc/apis/java/CMakeList.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+cmake_minimum_required(VERSION 3.14)
+project(mmdeploy_java)
+
+macro(mmdeploy_python_add_module name)
+    if (TARGET mmdeploy_${name})
+        list(APPEND MMDEPLOY_PYTHON_SRCS ${CMAKE_CURRENT_SOURCE_DIR}/${name}.cpp)
+    endif ()
+endmacro()
+
+mmdeploy_java_add_module(detector)
```

Comment:
Miss the definition of macro `mmdeploy_java_add_module`. 
If the target is a `Module`, how it can be used, by loading a dynamic library or others?